code/videoDrawer.py

```diff
@@ -216,35 +216,31 @@ def frameOf(p, offset):
 lastTimestamp = float(lastParts[0])
 FRAME_COUNT = timestepToFrames(lastTimestamp+1)
 phonemeTimeline = []
-for i in range(len(schedules[4])):
-    parts = schedules[4][i].split(",")
-    timestamp = float(parts[0])
-    framestamp = timestepToFrames(timestamp)
+for elem in schedules[4]:
+    timestamp, _, phoneme = elem.split(",")
+    framestamp = timestepToFrames(float(timestamp))
     if i >= 1 and framestamp <= phonemeTimeline[-1][1]: # we have a 0-frame phoneme! Try to fix it.
         if i >= 2 and phonemeTimeline[-2][1] <= framestamp-2:
             phonemeTimeline[-1][1] = framestamp-1 # shift previous one back
         else:
             framestamp += 1 # shift current one forward
-    phoneme = parts[2]
-    phonemeTimeline.append([phoneme,framestamp])
-phonemeTimeline.append(["end",FRAME_COUNT])
+    phonemeTimeline.append([phoneme, framestamp])
```

Comment:
`phoneme` is never set anymore

Reply:
Fix with `phoneme = parts[2]` as original had it